CI: L4LB: warning: unused variable 'buf'

Everytime the L4LB XDP workflow runs, the following warning is emitted during compilation:
```
Run ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh cilium b49ab9592821760c2cb8d1ba49b3f90a7a84de9f'"
+ IMG_OWNER=cilium
+ IMG_TAG=b49ab9592821760c2cb8d1ba49b3f90a7a84de9f
+ clang -O2 -Wall -target bpf -c bpf_xdp_veth_host.c -o bpf_xdp_veth_host.o
+ clang -O2 -Wall -target bpf -c test_tc_tunnel.c -o test_tc_tunnel.o
test_tc_tunnel.c:456:7: warning: unused variable 'buf' [-Wunused-variable]
        char buf[sizeof(struct v6hdr)];
             ^
1 warning generated.
```

--- a/test/l4lb/test_tc_tunnel.c
+++ b/test/l4lb/test_tc_tunnel.c
@@ -453,7 +453,6 @@ int __encap_ip6udp_eth(struct __sk_buff *skb)
 
 static int decap_internal(struct __sk_buff *skb, int off, int len, char proto)
 {
-	char buf[sizeof(struct v6hdr)];
 	struct gre_hdr greh;
 	struct udphdr udph;
 	int olen = len;

